### DISCUSSION TITLE
drag and drop events from all day to timeslot and vice versa

### DISCUSSION THREAD

**Discussion by rflannery** [vkurko/calendar#327] [Ideas]
Hello, thank you for the great and really useful library.

I was wondering is there any chance that you could implement dragging of events from the allday slot to a time slot and back.

Thank you in any case

Best regards

**Comment by vkurko** [comment]
Technically it is more possible than not. I will try to work on this feature in the future.

**Reply by rflannery** [reply]
Thank you, best regards
